shaping/input_test.go

```diff
@@ -0,0 +1,162 @@
+package shaping
```

Comment:
For the tests, what would you think about importing something like `gioui.org/font/gofont` and testing support of an actual font against some multi-lingual text? There's also `eliasnaur.com/roboto`. These test cases are excellent unit tests, but I think we could stand to have one or two tests that use a real font. It doesn't need to be the ones I mentioned; they would just be easy because they're importable packages.

Reply:
I also think we should add tests with a real font. 
Should we favor (open source) font files (to be loaded with textlayout/fonts) to avoid adding a dependency ? 
Would it be possible for you to provide some text input ? Otherwise I can try and build some multi-lingual text as you suggested.